seabird_chat_ingest.proto

```diff
@@ -0,0 +1,150 @@
+syntax = "proto3";
+
+option go_package = ".;pb";
+
+import "common.proto";
+
+package seabird;
+
+// Note that all of the types in this file end with ChatRequest, end with
+// ChatEvent, or start with Chat to prevent namespace conflicts between the main
+// seabird.proto.
+
+message ChatUser {
+  string id = 1;
+  string display_name = 2;
+}
+
+message ChatChannel {
+  string id = 1;
+  string display_name = 2;
+}
```

Comment:
Why not use the same messages as you have in `seabird.proto`? `Channel` has one additional field, but it seems weird to have these be separate.

Reply:
I have them separate because they have an additional field. That's exactly the reason. I'd rather not return a partially filled out message when possible.

Reply:
I see. Why doesn't `ChatChannel` have the `topic` field? And `ChatUser` and `User` have the same fields. Why shouldn't they have that extra data in chat backends?